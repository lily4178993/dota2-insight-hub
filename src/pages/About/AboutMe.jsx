import React from 'react';
import './aboutMe.css';

const AboutMe = () => (
  <div className="about-me-container">
    <h2 className="about-me-title">About Me</h2>
    <div className="about-me-content">
      <p>
        Hi there! My name is Nelly Telli, and I&apos;m a software engineer with a passion
        for web development and problem-solving. I love building responsive web
        applications and creating clean, maintainable code.
      </p>
      <br />
      <p>
        My favorite quote is,
        {' '}
        <strong>
          &ldquo;The only way to do great work is to love what you
          do&rdquo;
        </strong>
        {' '}
        by Steve Jobs. I strongly believe in the importance of loving your
        work and staying curious to keep learning and growing.
      </p>
    </div>
  </div>
);

export default AboutMe;
