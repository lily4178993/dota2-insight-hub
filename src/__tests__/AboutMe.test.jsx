import React from 'react';
import { render } from '@testing-library/react';
import AboutMe from '../pages/About/AboutMe';

test('AboutMe component matches snapshot', () => {
  const { asFragment } = render(<AboutMe />);
  expect(asFragment()).toMatchSnapshot();
});
