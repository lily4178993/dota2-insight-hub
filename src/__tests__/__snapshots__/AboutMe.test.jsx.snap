// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="about-me-container"
  >
    <h2
      class="about-me-title"
    >
      About Me
    </h2>
    <div
      class="about-me-content"
    >
      <p>
        Hi there! My name is Nelly Telli, and I'm a software engineer with a passion for web development and problem-solving. I love building responsive web applications and creating clean, maintainable code.
      </p>
      <br />
      <p>
        My favorite quote is, 
        <strong>
          “The only way to do great work is to love what you do”
        </strong>
         by Steve Jobs. I strongly believe in the importance of loving your work and staying curious to keep learning and growing.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
